hip-more-cultural-names/common/landed_titles/0_HIP_MoreCulturalNames.txt

```diff
@@ -15500,12 +15422,7 @@ e_byzantium = {
                 b_melnik_strumica = {
                 }
                 b_stip = {
-                    bosnian = "�tip"
-                    bulgarian = "�tip"
-                    carantanian = "�tip"
-                    croatian = "�tip"
```

Comment:
again its about the š

Reply:
Thanks. I removed them by mistake. I did some replacements and I must have missed those diacritics